openedx/core/djangoapps/programs/utils.py

```diff
@@ -284,6 +286,51 @@ def completed_programs(self):
         """
         return [program['uuid'] for program in self.programs if self._is_program_complete(program)]
 
+    @property
+    def completed_programs_with_available_dates(self):
```

Comment:
Since we're switching to this, does it make since to get rid of completed_programs at https://github.com/edx/edx-platform/pull/18862/files#diff-1072896d9c36e00edfdd4998d9e13d22R281 ?  I don't think I see it being used anywhere else

Reply:
Right you are. I didn't think to confirm if it was still used elsewhere. I'll remove it.